.circleci/config.yml

```diff
@@ -51,6 +51,14 @@ jobs:
           root: ~/grommet-ci
           paths:
             - ./
+  bundlesize:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/grommet-ci
+      - run:
+          name: size
+          command: yarn bundlesize
```

Comment:
Is this supposed to be `yarn test-bundlesize`, since that's the scripts key?

Reply:
lol, this was indeed a typo, but why did it work on the ci is weird!

Reply:
maybe because there is a bundlesize script in .bin?